package.json

```diff
@@ -40,7 +40,7 @@
     "through2": "^0.6.3",
     "typescript": "1.8.0",
     "validator": "^3.33.0",
-    "vsts-task-lib": "^0.5.6"
+    "vsts-task-lib": "^0.6.2"
```

Comment:
vsts-task-lib don't remove functionality or change. 99.99% it shouldn't. Without this version, this task won't run.